DFE-Digital/early-years-foundation-recovery
# Er 342 summative assessment system spec coverage

Ticket [ER-342](https://dfedigital.atlassian.net/browse/ER-342)

- Add specs to test summative assessment
- Add specs to test formative assessment
- Add specs to test confidence check
